esmvaltool/recipes/recipe_flato13ipcc/recipe_flato13ipcc_figures_938_941.yml

```diff
@@ -0,0 +1,293 @@
+                 and mumltimodel over the selected project&experiemnt&mip
+    themes:
+      - chem
+    realms:
+      - atmos
+    variables:
+      tas:
+        preprocessor: annual_cycle
+        reference_dataset: ERA-Interim
+        mip: Amon
+    additional_datasets:
+      - {dataset: ERA-Interim, project: OBS6, type: reanaly, version: 1,
+         start_year: 1986, end_year: 2005, tier: 3}
+    scripts:
+      clim: &Fig938_settings
+        script: regional_downscaling/Figure9.38.ncl
+        styleset: CMIP5
+        fig938_region_label: ["WNA", "ENA", "CAM", "TSA", "SSA", "EUM", "NAF",
+                              "CAF", "SAF", "NAS", "CAS", "EAS", "SAS", "SEA",
+                              "AUS"]
+        fig938_project_MMM: ["CMIP5"]
+        fig938_experiment_MMM: ["historical"]
+        fig938_mip_MMM: ["Amon"]
+        fig938_colors_MMM: ["red"]
+        fig938_refModel: ERA-Interim
+        fig938_MMM: true
+
+  # ==========================================================================
+  # Chapter 9  - Figure939
+  # ==========================================================================
+  regional_downscaling_Fig939:
+    description: Creates seasonal or annual bias box plots at selected
+                 regions over the selected project&experiemnt&mip over
+                 the selected project&experiemnt&mip
+    themes:
+      - chem
+    realms:
+      - atmos
+    variables:
+      tas:
+        preprocessor: clima
+        reference_dataset: ERA-Interim
+        mip: Amon
+    additional_datasets:
+      - {dataset: ERA-Interim, project: OBS6, type: reanaly, version: 1,
+         start_year: 1986, end_year: 2005, tier: 3}
+
+    scripts:
+      clim: &Fig939_settings
+        script: regional_downscaling/Figure9.39.ncl
+        styleset: CMIP5
+        fig939_season: ["DJF", "JJA", "ANN"]
+        fig939_region_label: ["ALAs", "CGIs", "WNAs", "CNAs", "ENAs", "CAMs",
+                              "AMZs", "NEBs", "WSAs", "SSAs", "NEUs", "CEUs",
+                              "MEDs", "SAHs", "WAFs", "EAFs", "SAFs", "NASs",
+                              "WASs", "CASs", "TIBs", "EASs", "SASs", "SEAs",
+                              "NAUs", "SAUs"]
+        fig939_project_MMM: ["CMIP5"]
+        fig939_experiment_MMM: ["historical"]
+        fig939_mip_MMM: ["Amon"]
+        fig939_MMM: true
+
+  # ==========================================================================
+  # Chapter 9  - Figure940
+  # ==========================================================================
+  regional_downscaling_Fig940:
+    description: Creates annual cycle regional plots for individual models
+                 and mumltimodel over the selected project&experiemnt&mip
+    themes:
+      - chem
+    realms:
+      - atmos
+    variables:
+      pr:
+        preprocessor: clima_nomask
+        reference_dataset: CRU
+        mip: Amon
+    additional_datasets:
```

Comment:
The diagnostic can only use one reference data set at once, so only one observational data set should be included (else the second one would become part of the multi-model mean). ERA-Interim was included and commented out to mark it as potential alternative instead of CRU. In the IPCC AR5 ERA-Interim is used for temperature for the ocean regions (Fig. 9.40), GPCP for precipitation over the ocean regions (which is not included, I'm not sure if it was available when Irene wrote the recipe). CRU is used in Fig 9.39  (land regions) of the IPCC AR5 for both temperature and precipitation. The choice of data sets here is the one that was used in the ESMValTool paper (https://gmd.copernicus.org/articles/14/3159/2021/), see also comment, which I'll add to Axels question later. 
I could remove the commented data sets or leave them commented?